server/db/index.js

```diff
@@ -46,6 +47,10 @@ Comment.belongsTo(User);
 Provider.hasMany(Course);
 Course.belongsTo(Provider);
 
+Course.belongsTo(Topic);
+Post.belongsTo(Topic);
+Event.belongsTo(Topic);
```

Comment:
Do we need to associate Topics with Posts/Events? I thought we would associate the Topics with Courses and Groups. And Posts/Events just be directly associated with Groups

Reply:
Maybe I'm confused about what we mean by topic

Reply:
True. Maybe I am the one that is confused. Apologies. I read into the post and event topic field, and misunderstood. I think you are right and will fall back and fix those associations. 

Changing them to be associated to group and course. Oops